commands/Music/clearqueue.js

```diff
@@ -0,0 +1,32 @@
+const { Command } = require("klasa");
+
+module.exports = class extends Command {
+
+    constructor(...args) {
+        super(...args, {
+            runIn: ["text"],
+            cooldown: 10,
+            aliases: ["cq", "prunequeue"],
+            permissionLevel: 0,
+            requiredPermissions: ["USE_EXTERNAL_EMOJIS"],
+            description: (msg) => msg.language.get("COMMAND_CLEARQUEUE_DESCRIPTION"),
+            extendedHelp: "No extended help available."
+        });
+        this.Music = true;
```

Comment:
shouldn't start with a capital and should be called like requireMusic or something like that